packages/@react-aria/slider/src/useSlider.ts

```diff
@@ -168,7 +171,7 @@ export function useSlider(
     }
   };
 
-  if (labelProps.htmlFor) {
+  if ('htmlFor' in labelProps && labelProps.htmlFor) {
```

Comment:
what is this change about? wouldn't `labelProps?.htmlFor` work?

Reply:
The type returned by `useLabel` is `DOMAttributes | LabelHTMLAttributes<HTMLLabelElement>`, depending on the `labelElementType`. If the label is not a `<label>` element, then the type won't have an `htmlFor` prop, so TS requires this check.